src/log.c

```diff
@@ -138,6 +138,7 @@ static size_t generateEntryHeader(raft_entry_t *ety, unsigned char *buf, size_t
     pos += multibulkWriteLong(pos, end - pos, ety->id);
     pos += multibulkWriteUInt64(pos, end - pos, ety->session);
     pos += multibulkWriteInt(pos, end - pos, ety->type);
+
```

Comment:
```suggestion
```
Is this change intentional? If not, let's get rid of it.

Reply:
it sort of was.  the ety-> entries are the raft_entry_t header (id/session/type), the next entry really belong with the data (i.e. we have data_len here, but data is appended separately) which gets appended externally to this, so wanted to make it clear.

Reply:
Change is okay but probably it'd be nicer as part of another PR. As this is the only change in log.c, when you look that this PR later or to the commit history, it'll give you an impression that we changed log.c as part of this PR.